### PR TITLE
Components/box

### PR DESCRIPTION
**Pull request by binhtt13122000**

### CODE REVIEW

File: src/components/Boxs/BoxBase/index.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ const BoxBase: React.FC<BoxBaseProps> = ({
         color = color || "text.primary";
     }
     if (overrideColor) {
-        color = overrideColor;
+        color = overrideColor + ".main";
```
Comment (binh1298) [inline src/components/Boxs/BoxBase/index.tsx:119]: I don't understand the logic behind this

Comment (binh1298) [reply src/components/Boxs/BoxBase/index.tsx:119]: Shouldn't the color be .contrastText?

Comment (binhtt13122000) [reply src/components/Boxs/BoxBase/index.tsx:119]: When you choosing a bgColor (example secondary), it's color is secondary.constractText automatically.

Comment (binhtt13122000) [reply src/components/Boxs/BoxBase/index.tsx:119]: I make it for Kien's task. In his task about List, he must using bgColor is primary but color is secondary. So I make overrideColor for exceptions. If you don't using overrideColor, color is always onA when bgColor is A.

Comment (binhtt13122000) [reply src/components/Boxs/BoxBase/index.tsx:119]: image

**Review by binh1298 — APPROVED** [review]

**Pull request merged**